app/models/skill.rb

```diff
@@ -1,7 +1,7 @@
 class Skill < ApplicationRecord
   SKILLS = ['Maths', 'English', 'Physics', 'Chemistry', 'Biology', 'Geography', 'History', 'Programming']
 
-  belongs_to :volunteer
+  belongs_to :user
```

Comment:
think this should be :volunteer still

Reply:
yeah me too - I had to change it to align with an earlier schema change